src/transforms/transform-pull-request.ts

```diff
@@ -30,7 +30,7 @@ const mapReviews = async (
 	gitHubInstallationClient: GitHubInstallationClient
 ): Promise<JiraReview[]> =>
 {
-	const sortedReviews = orderBy(reviews, "submitted_at", "desc");
+	const sortedReviews = orderBy(reviews, "submitted_at", "asc");
```

Comment:
Basically there were 2 issues. When GitHub sends all the review data we were essentially reversing the order so we were acknowledging the first review state for a reviewer, not the latest.